### ISSUE TITLE
Shift based batch normalization

### ISSUE THREAD

**Issue by jack-kjlee** [itayhubara/BinaryNet.pytorch#12]
Is there any implementation of shift based batch normalization in Pytorch version of BinaryNet?
